src/metabase/fingerprinting.clj

```diff
@@ -0,0 +1,458 @@
+(ns metabase.fingerprinting
+  "Fingerprinting (feature extraction) for various models."
+  (:require [bigml.histogram.core :as hist]
+            [bigml.sketchy.hyper-loglog :as hyper-loglog]
+            [clj-time.coerce :as t.coerce]
+            [clj-time.core :as t]
+            [clj-time.format :as t.format]
+            [clj-time.periodic :as t.periodic]
+            [kixi.stats.core :as stats]
+            [kixi.stats.math :as math]
+            [medley.core :as m]
+            [metabase.db.metadata-queries :as metadata]
+            [metabase.models.card :refer [Card]]
+            [metabase.models.field :refer [Field]]
+            [metabase.models.segment :refer [Segment]]
+            [metabase.models.table :refer [Table]]
+            [redux.core :as redux]
+            [tide.core :as tide]))
```

Comment:
the style we've been following elsewhere in the codebase is to group these things together, Emacs will do it for you if you run `M-x cljr-clean-ns`. Make sure you `(setq cljr-favor-prefix-notation t)`